frontend/public/module/k8s/get-resources.ts

```diff
@@ -79,21 +80,7 @@ export const getCachedResources = () =>
     resolve(null);
   });
 
-export type DiscoveryResources = {
-  adminResources: string[];
-  allResources: string[];
-  configResources: K8sKind[];
-  clusterOperatorConfigResources: K8sKind[];
-  models: K8sKind[];
-  namespacedSet: Set<string>;
-  safeResources: string[];
-  groupVersionMap: {
-    [key: string]: {
-      versions: string[];
-      preferredVersion: string;
-    };
-  };
-};
+export { DiscoveryResources } from '@console/dynamic-plugin-sdk/src/api/common-types';
```

Comment:
not sure why, but am seeing a warning on the terminal on `yarn dev`. Can we import directly as it's just 3 files where it's being used and imported via this reexport?

![image](https://user-images.githubusercontent.com/5129024/148400764-f507aa00-f2f9-4d3b-b968-fb786f7977c7.png)

Reply:
@invincibleJai good catch. I thought this was a problem with my dev environment. I've made the suggested changes and everything works now with no warnings in dev mode!